CHANGELOG.md

```diff
@@ -18,6 +18,7 @@
 + Exported `@algorand-builder/algob`, `@algorand-builder/runtime` error types and make it accessible for API documentation.
 + Added `debugStack` option in `runtime.executeTx()` to print stack (upto depth = debugStack) after each opcode execution.
 + Tealv3 support in `@algo-builder/runtime`: `getBit`, `setBit`
++ Transpile TEAL code to substitute the TMPL placeholders
```

Comment:
Should we add this in **##Breaking** section? What happens if user is already using `TMPL_` prefixes in their `.py` contracts (like we were using before)?

Reply:
not sure if it's API breaking changes

Reply:
these are breaking though